src/component/ui/card/LectureCard.tsx

```
@@ -0,0 +1,37 @@
+    {tags && tags.length > 0 && (
+      <div className="card-info-lecture-tags">
+        {tags.map((tag, i) => (
+          <span key={i} className="card-info-lecture-tag">
+            {tag}
+          </span>
+        ))}
+      </div>
+    )}
+    <div>
+      {
+        description && description.map(des => <p className="card__description">{des}</p>)
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)